src/common/state/error_table.h

```diff
@@ -4,12 +4,26 @@
 #include "db.h"
 #include "table.h"
 
+/// An ErrorIndex may be used as an index into error_types.
+enum class ErrorIndex : int32_t {
+  /// An object was added with a different hash from the existing one.
+  OBJECT_HASH_MISMATCH_ERROR_INDEX = 0,
```

Comment:
Let's get rid of the `ERROR_INDEX` suffix now that we have the `ErrorIndex::` prefix